saleor/graphql/csv/mutations/export_voucher_codes.py

```diff
@@ -0,0 +1,113 @@
+import graphene
+from django.core.exceptions import ValidationError
+
+from ....csv import models as csv_models
+from ....csv.error_codes import ExportErrorCode
+from ....csv.events import export_started_event
+from ....csv.tasks import export_voucher_codes_task
+from ....permission.enums import DiscountPermissions
+from ....webhook.event_types import WebhookEventAsyncType
+from ...app.dataloaders import get_app_promise
+from ...core import ResolveInfo
+from ...core.descriptions import ADDED_IN_318, PREVIEW_FEATURE
+from ...core.doc_category import DOC_CATEGORY_DISCOUNTS
+from ...core.types import BaseInputObjectType, ExportError, NonNullList
+from ...core.utils import WebhookEventInfo, raise_validation_error
+from ...core.validators import validate_one_of_args_is_in_mutation
+from ..enums import FileTypeEnum
+from .base_export import BaseExportMutation
+
+
+class ExportVoucherCodesInput(BaseInputObjectType):
+    voucher_id = graphene.GlobalID(
+        required=False,
+        description=(
+            "The ID of the voucher. "
+            "If provided, exports all codes belonging to the voucher."
+        ),
+    )
+    ids = NonNullList(
+        graphene.ID,
+        description="List of voucher code IDs to export.",
+        required=False,
+    )
+    file_type = FileTypeEnum(description="Type of exported file.", required=True)
+
+    class Meta:
+        doc_category = DOC_CATEGORY_DISCOUNTS
+
+
+class ExportVoucherCodes(BaseExportMutation):
+    class Arguments:
+        input = ExportVoucherCodesInput(
+            required=True, description="Fields required to export voucher codes."
+        )
+
+    class Meta:
+        description = (
+            "Export active voucher codes to csv/xlsx file."
```

Comment:
`active voucher codes` - is it correct? Currently csv logic exports `all()` codes. without checking `is_active` flag on VoucherCode model.

Reply:
yes this is misleading information, we should not use `active` word in description